static/js/TopicLandingPage/TopicSalad.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {useEffect, useState} from "react";
+import {WordSalad} from "../WordSalad";
+import Sefaria from "../sefaria/sefaria";
+import {InterfaceText} from "../Misc";
+
+
+export const TopicSalad = () => {
+
+    const [salad, setSalad] = useState([]);
+    const lang = Sefaria.interfaceLang == 'hebrew' ? 'he' : 'en';
+
+    const renderSaladItem = (item) => {
+        return(<a href={`/topics/${item.slug}`} className="topic-salad-item">
+                <InterfaceText text={{en: item.text, he: item.text}}/>
+                </a>)
+    }
+
+    const fetchRandomSaladItems = async () => {
+        const poolName = Sefaria.getLangSpecificTopicPoolName('general', lang);
+        const topics = await Sefaria.getTopicsByPool(poolName, 50);
+        const saladItems = topics.map(topic=>({slug: topic.slug, text: topic.primaryTitle[lang]}))
```

Comment:
lang is not needed here since you're using InterfaceText to render the items. Instead make the object `{slug: topic.slug, text: topic.primaryTitle}` and the when rendering the item `<InterfaceText text={item.text} />`. This is much cleaner. 

Note, I understand the reason this was done was to allow WordSalad to replace spaces with non-breaking spaces. see comment below on using `nowrap` in CSS.